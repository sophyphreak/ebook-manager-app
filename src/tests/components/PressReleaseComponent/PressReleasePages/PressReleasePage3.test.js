import React from 'react';
import { shallow } from 'enzyme';
import PressReleasePage3 from '../../../../components/PressReleaseComponent/PressReleasePages/PressReleasePage3';

let error, wrapper;

beforeEach(() => {
  error = {
    message: ''
  };
  wrapper = shallow(<PressReleasePage3 error={error} />);
});

test('should render PressReleasePage3 correctly', () => {
  expect(wrapper).toMatchSnapshot();
});
