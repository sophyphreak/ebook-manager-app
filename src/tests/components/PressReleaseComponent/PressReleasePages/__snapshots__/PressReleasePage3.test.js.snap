// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PressReleasePage3 correctly 1`] = `
<Card
  className="animated fadeIn"
  tag="div"
>
  <CardHeader
    tag="div"
  >
    <h4>
      <strong>
        Press Release - 3 of 3
      </strong>
    </h4>
  </CardHeader>
  <CardBody
    tag="div"
  >
    <Form
      tag="form"
    >
      <Website
        hasError={false}
        label="Website"
      />
      <Keywords
        hasError={false}
        label="Keywords"
      />
      <ReleaseText
        hasError={false}
        label="Press Release*"
      />
      <Button
        color="basic"
        size="sm"
        tag="button"
      >
        Go Back
      </Button>
      <Button
        color="primary"
        size="sm"
        tag="button"
        type="submit"
        value="Submit"
      >
        Next Section
      </Button>
    </Form>
  </CardBody>
</Card>
`;
